docs/installing.rst

```diff
@@ -0,0 +1,31 @@
+Installing Thumbor Community Shortener
+======================================
+
+1. Install thumbor (see `Thumbor repository`_)
+2. Install `Thumbor Community Core`_
+3. Clone this repository.
+4. If you've set a virtualenv up for thumbor, activate it.
+5. Install the Thumbor Community Shortener:
+::
+    $ cd thumbor-community/shortener
+    $ pip install .
+
+6. Register the extension within Thumbor's configuration file:
+::
+    COMMUNITY_EXTENSIONS = [
+        'shortener',
+        ...
+    ]
+
+7. Configure the extension within the same file:
+::
+    SHORTENER_GENERATOR = 'Generator'   # Shortener generator class name, defaults to Generator
```

Comment:
Should be something like:
```
SHORTENER_STORAGE='tc_shortener.storages.redis_storage'
SHORTENER_GENERATOR='tc_shortener.generators.sha256_generator'
```

The configuration is the same as the RESULT_STORAGE thumbor keys.